### ISSUE TITLE
Android app crashes after trying to resume video recording

### ISSUE THREAD

**Issue by ridvanaltun** [ridvanaltun/react-native-deepar#2]
Android app crashes when resuming video recording after pausing

**Comment by ridvanaltun** [comment]
Android logs:

```
E/MPEG4Writer: do not support out of order frames (timestamp: 3329724 < last: 6027703 for Audio track
E/MPEG4Writer: 0 frames to dump timeStamps in Audio track 
I/MPEG4Writer: Received total/0-length (140/0) buffers and encoded 139 frames. - Audio
I/MPEG4Writer: Audio track drift time: 0 us
I/timestamp: 163365533514
E/MediaAdapter: pushBuffer called before start
I/timestamp: 163365561098
E/AndroidRuntime: FATAL EXCEPTION: MediaAudioEncoder
    Process: com.example.reactnativedeepar, PID: 30565
    java.lang.IllegalStateException: writeSampleData returned an error
        at android.media.MediaMuxer.nativeWriteSampleData(Native Method)
        at android.media.MediaMuxer.writeSampleData(MediaMuxer.java:694)
        at ai.deepar.ar.MediaMuxerWrapper.writeSampleData(MediaMuxerWrapper.java:166)
        at ai.deepar.ar.MediaEncoder.drain(MediaEncoder.java:383)
        at ai.deepar.ar.MediaEncoder.run(MediaEncoder.java:167)
        at java.lang.Thread.run(Thread.java:919)
I/timestamp: 163365582149
W/ActivityThread: SCHED: com.example.reactnativedeepar/.MainActivity [85, r=2090ms, a=365ms, w=24906ms]
I/timestamp: 163365606856
I/Process: Sending signal. PID: 30565 SIG: 9
```

**Comment by ridvanaltun** [comment]
The issue fixed in Android SDK version 3.4.4: https://docs.deepar.ai/deepar-sdk/deep-ar-sdk-for-android/changelog#v344

I will upgrade the DeepAR SDK to resolve this issue.
